### PR TITLE
S02 issue23 header home page

### PR DESCRIPTION
**Pull request by val-fr69**
Add Header Home Page
Add Tag List under Header

### CODE REVIEW

**Review by l00ma — APPROVED** [review]

File: assets/styles/app.scss [hunk 1]
```diff
@@ -27,16 +27,112 @@ $animCurveEaseOut: cubic-bezier(.1, 1, .8, .8);
...
-    --plyr-control-icon-size: 20px;
+.home-title {
+    height: 40px;
+    width: 80vw;
+    font-size: $h4-font-size;
+}
 
-    --plyr-video-control-color: #ffffff;
+.header-logo {
+    background-color: $dark;
```
Comment (maxdlr) [inline assets/styles/app.scss:67]: Should be done with Bootstrap.
bg-dark

File: assets/styles/app.scss [hunk 2]
```diff
@@ -27,16 +27,112 @@ $animCurveEaseOut: cubic-bezier(.1, 1, .8, .8);
...
+    font-size: 45px;
```
Comment (maxdlr) [inline assets/styles/app.scss:68]: Should be done with Bootstrap.
fs-1,2,3,4,5

File: assets/styles/app.scss [hunk 3]
```diff
@@ -27,16 +27,112 @@ $animCurveEaseOut: cubic-bezier(.1, 1, .8, .8);
...
-    --plyr-video-control-color-hover: #FFE500;
+.button-view {
+    min-height: 300px;
+    margin-bottom: 20px;
+    z-index: 1;
+
+    .link-desktop {
+        background: $primary !important;
+        font-size: $h3-font-size;
+        text-decoration: none;
```
Comment (maxdlr) [inline assets/styles/app.scss:80]: Should be done with Bootstrap.
text-decoration-none

File: assets/styles/app.scss [hunk 4]
```diff
@@ -27,16 +27,112 @@ $animCurveEaseOut: cubic-bezier(.1, 1, .8, .8);
...
+        width: 200px;
+        padding: 5px;
+
+
+        .view {
+            margin-left: 20px;
```
Comment (maxdlr) [inline assets/styles/app.scss:86]: Should be done with Bootstrap.
ms-1,2,3,4,5

File: assets/styles/app.scss [hunk 5]
```diff
@@ -27,16 +27,112 @@ $animCurveEaseOut: cubic-bezier(.1, 1, .8, .8);
...
+        }
+
+        .icon {
+            margin-right: 10px;
+            font-size: 35px;
```
Comment (maxdlr) [inline assets/styles/app.scss:91]: Should be done with Bootstrap.
fs-1,2,3,4,5

File: assets/styles/app.scss [hunk 6]
```diff
@@ -27,16 +27,112 @@ $animCurveEaseOut: cubic-bezier(.1, 1, .8, .8);
...
+
+        }
+    }
+
+    .link-mobile {
+        background: $primary !important;
+        font-size: $h6-font-size;
+        text-decoration: none;
+        width: calc(0.8*200px);
+        padding: 6px 3px;
```
Comment (maxdlr) [inline assets/styles/app.scss:102]: Should be done with Bootstrap.
bg-primary
fs-6
text-decoration-none
//
py-1,2,3,4,5
px-1,2,3,4,5

File: assets/styles/app.scss [hunk 7]
```diff
@@ -27,16 +27,112 @@ $animCurveEaseOut: cubic-bezier(.1, 1, .8, .8);
...
+
+
+        .view {
+            margin-left: calc(0.8*20px);
+            margin-bottom: 1px;
```
Comment (maxdlr) [inline assets/styles/app.scss:107]: Should be done with Bootstrap.
ms-1,2,3,4,5
mb-1,2,3,4,5

File: assets/styles/app.scss [hunk 8]
```diff
@@ -27,16 +27,112 @@ $animCurveEaseOut: cubic-bezier(.1, 1, .8, .8);
...
+            font-size: 19px;
+            margin-left: calc(1.2*10px);
+
+        }
+    }
+}
+
+.header-infos-desktop {
+    min-height: 300px;
+    z-index: 1;
```
Comment (maxdlr) [inline assets/styles/app.scss:121]: Should be done with Bootstrap.
z-1

File: assets/styles/app.scss [hunk 9]
```diff
@@ -27,16 +27,112 @@ $animCurveEaseOut: cubic-bezier(.1, 1, .8, .8);
...
+
+    .title {
+        font-size: $h4-font-size;
+        margin-bottom: 20px;
```
Comment (maxdlr) [inline assets/styles/app.scss:125]: Should be done with Bootstrap.
fs-4
mb-1,2,3,4,5

File: assets/styles/app.scss [hunk 10]
```diff
@@ -27,16 +27,112 @@ $animCurveEaseOut: cubic-bezier(.1, 1, .8, .8);
...
+    }
+}
+
+.header-infos-mobile {
+    min-height: 300px;
+    z-index: 1;
```
Comment (maxdlr) [inline assets/styles/app.scss:131]: Should be done with Bootstrap.
z-1

File: assets/styles/app.scss [hunk 11]
```diff
@@ -27,16 +27,112 @@ $animCurveEaseOut: cubic-bezier(.1, 1, .8, .8);
...
+
+    .title {
+        font-size: $h5-font-size;
```
Comment (maxdlr) [inline assets/styles/app.scss:134]: Should be done with Bootstrap.
fs-5

File: assets/styles/app.scss [hunk 12]
```diff
@@ -303,3 +399,44 @@ body {
     }
 }
 
+.load-more-tag {
+    font-family: $textFont;
+    font-size: $h6-font-size;
```
Comment (maxdlr) [inline assets/styles/app.scss:404]: Should be done with my custom class.
text-font
Should be done with Bootstrap.
fs-6

File: assets/styles/app.scss [hunk 13]
```diff
@@ -303,3 +399,44 @@ body {
...
+        scale: 99%;
+        background-color: $light !important;
+        color: $dark !important;
+    }
+
+}
+
+.load-more-tag-mobile {
+    font-family: $textFont;
+    font-size: $h6-font-size;
```
Comment (maxdlr) [inline assets/styles/app.scss:425]: Should be done with my custom class.
text-font
Should be done with Bootstrap.
fs-6

**Review by maxdlr — CHANGES_REQUESTED** [review]
Désolé fréro, mais faut que t'utilise Bootstrap au max et n'utiliser le CSS que pour ce qu'il ne propose pas, ou les hover.. etc

**Review by maxdlr — APPROVED** [review]

**Review by l00ma — APPROVED** [review]

File: templates/video/show.html.twig [hunk 14]
```diff
@@ -26,7 +26,11 @@
 					<div class="sh-hash d-lg-flex d-none col-lg-auto justify-content-end align-items-start">{{ include('shared/category-tag.html.twig', {tag: video.tag[0].name , style: 'tag-main'}) }}</div>
 				</div>
 
+<<<<<<< HEAD
+				<div class="sh-player video-container rounded-4 m-4">
+=======
 				<div class="sh-player video-container m-md-4 my-2 mx-0">
+>>>>>>> 54353524cc3ed09dd97923aa08436164cc61a34c
```
Comment (aldaitzwild) [inline templates/video/show.html.twig:33]: Unresolved conflicts !

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**